ci-operator/jobs/openshift/assisted-image-service/openshift-assisted-image-service-main-presubmits.yaml

```diff
@@ -1,5 +1,81 @@
 presubmits:
   openshift/assisted-image-service:
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^main$
+    - ^main-
+    cluster: build04
+    context: ci/prow/e2e-metal-assisted
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/cloud: packet-edge
+      ci-operator.openshift.io/cloud-cluster-profile: packet-assisted
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-assisted-image-service-main-e2e-metal-assisted
+    optional: true
+    rerun_command: /test e2e-metal-assisted
+    skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|OWNERS_ALIASES|PROJECT|LICENSE)$
```

Comment:
I think it means it will still run if changes are not fully compatible with this regex